9.2/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -114,6 +114,19 @@ function generate_postgresql_recovery_config() {
       > "${POSTGRESQL_RECOVERY_FILE}"
 }
 
+# this function iterates over hooks.d and seek for folders
+# which contains post-init executables
+function execture_post_init_scripts() {
+  for i in ${CONTAINER_SCRIPTS_PATH}/hooks.d/*
```

Comment:
i wonder if hooks.d should be renamed to post-init-hooks.d, because what if you later want to introduce a pre-init hook option?